### ISSUE TITLE
no separators causes glitch

### ISSUE THREAD

**Issue by shiruken1** [jonschlinkert/time-stamp#7]
Hello there!

Just thought I'd report that if I run:

```
fs.createWriteStream('test/output_'+timestamp('YYYYMMDDss')+'.png')
```
I get:
```
output_2017MMDD05.png
```

**Comment by jonschlinkert** [comment]
Is this a feature request? Or is there something in the unit tests or documentation that led you to believe that format is supported?

**Comment by Avaray** [comment]
Same "issue" here. I use this format of date very often (mostly for naming files).

It would be really nice to be able to use this:
`timestamp('YYYYMMDD')`
instead of this:
`timestamp('YYYY') + timestamp('MM') + timestamp('DD');`

**Comment by jonschlinkert** [comment]
Fixed in 1.1.0. thanks for the comment/ping
